README.md

```diff
@@ -137,7 +139,7 @@ Optionally, register the `FacesServlet` in a Servlet 5.0+ compatible deployment
 
 Noted should be that Jakarta Faces 3.0+ is already "implicitly" registered and mapped on `*.jsf`, `*.faces` and `/faces/*` when running on a Jakarta Servlet 5.0+ container. This will be overridden altogether when explicitly registering as above. [The `*.xhtml` URL pattern is preferred over above for security and clarity reasons][17]. Jakarta Faces 2.3+ adds `*.xhtml` to set of default patterns, hence the `FacesServlet` registration being optional. But when you don't explicitly map it on `*.xhtml`, then people can still access JSF pages using `*.jsf`, `*.faces` or `/faces/*` URL patterns. This is not nice for SEO as JSF by design doesn't 301-redirect them to a single mapping.
 
-The Faces deployment descriptor file `/WEB-INF/faces-config.xml` is fully optional, but if any it must be Jakarta Faces 3.0 compatible, otherwise Jakarta Faces 3.0 will run in a fallback modus matching the exact `version` as declared in `<faces-config>` root element.
+The Faces deployment descriptor file `/WEB-INF/faces-config.xml` is fully optional, but if any it must be Jakarta Faces 3.0+ compatible, otherwise Jakarta Faces 4.0 will run in a fallback modus matching the exact `version` as declared in `<faces-config>` root element.
```

Comment:
Is this correct?

Reply:
Nope, the `3.0+` part should also say `4.0`.